src/pages/steps/LocationStep.tsx

```diff
@@ -458,6 +481,32 @@ const LocationStep = ({
                       width="30%"
                       minHeight={parseSpacing(7)}
                     />
+                    {isCultuurkuurFeatureFlagEnabled && isCultuurkuurEvent && (
+                      <ToggleBox
+                        onClick={() =>
+                          onFieldChange({
+                            country: undefined,
+                            municipality: undefined,
+                            isOnline: false,
+                          })
+                        }
+                        active={
+                          municipality?.zip
+                            ? municipality.zip === '0000' && !isOnline
+                            : !country && !isOnline && !municipality
+                        }
```

Comment:
This active has a lot of nested tenary operators.
Is it possible to make it a bit more readable?